### PR TITLE
Return redemand in every handle_process

### PR DESCRIPTION
**Pull request by FelonEkonom**
Issue with a description, what is done in this PR: https://github.com/membraneframework/membrane_audio_mix_plugin/issues/35

### CODE REVIEW

**Comment by mat-hek** [comment]
Isn't this the hack that doesn't work for offline use cases? :P

**Comment by FelonEkonom** [comment]
> Isn't this the hack that doesn't work for offline use cases? :P

Maybe little bit 😉

**Comment by mat-hek** [comment]
ok

File: lib/membrane_audio_mixer.ex [hunk 1]
```diff
@@ -320,11 +321,10 @@ defmodule Membrane.AudioMixer do
     )
   end
 
-  defp mix_or_redemand(size, time_frame, context, state) do
+  defp mix_and_redemand(size, time_frame, context, state) do
     if size >= time_frame do
       {actions, state} = mix_and_get_actions(context, state)
-      actions = if actions == [], do: [redemand: :output], else: actions
-      {actions, state}
+      {actions ++ [redemand: :output], state}
```
Comment (Karolk99) [inline lib/membrane_audio_mixer.ex:327]: We can leave it like that or redemand only when mixed audio's byte size is smaller than the input audio's size.

**Review by Karolk99 — APPROVED** [review]

**Pull request merged**